internal/provider/models/destinations/test/provider_test.go

```diff
@@ -0,0 +1,42 @@
+package destinations
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-framework/providerserver"
+	"github.com/hashicorp/terraform-plugin-go/tfprotov6"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/mezmo/terraform-provider-mezmo/internal/provider"
+	"github.com/mezmo/terraform-provider-mezmo/internal/provider/providertest"
+)
+
+var testAccProtoV6ProviderFactories = map[string]func() (tfprotov6.ProviderServer, error){
+	"mezmo": providerserver.NewProtocol6WithError(provider.New("destinations_test")()),
+}
+
+// testAccBackend checks that userConfig matches whats is in the
+// backend for resourceName
```

Comment:
I had asked a question about this, but maybe you didn't see it.  Why are we testing what's in the BE database?  If we're doing the provider right, the values we put in test assertions should match state and the api responses.  I don't think it should be this provider's job to check the service BE directly.  We should trust the api responses.

Reply:
Ah, I just realized that the file consolidation doesn't use NullifyPlanFields, which protects us from the BE not updating.  Is that why this function was added?